app/views/orders/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Your Orders</h2>
```

Comment:
Do we want to consider sectioning the show page into current orders, completed orders, cancelled orders, etc.?